example/requirements.txt

```diff
@@ -1 +1,3 @@
-Django>=1.8
\ No newline at end of file
+django-environ==0.4.0
```

Comment:
I don't think introducing this into the example is worth the indirection it adds.